### PR TITLE
Merge visible-emotions into main

### PR DESCRIPTION
**Pull request by ATY02**
Everyone in class said they liked what they saw. I suggest we leave the branch open for the Chatterbot emotions.

### CODE REVIEW

**Review by will-hoover — COMMENTED** [review]
Can we make sure the tests pass before merging this?

File: frontend/src/pages/Chat.jsx [hunk 1]
```diff
@@ -87,66 +103,93 @@ const Chat = () => {
         }
     };
 
+    useEffect(() => {
+        let img = 5;
+        console.log(chatHistory);
```
Comment (will-hoover) [inline frontend/src/pages/Chat.jsx]: What is the console.log for?

**Review by will-hoover — APPROVED** [review]
Changes look good, I love the updates.

**Comment by ATY02** [comment]
I have masterfully removed conflicts, if anyone wants to check this branch.

File: frontend/src/pages/Chat.jsx (hunk 1 above)
Comment (Masonk31) [reply frontend/src/pages/Chat.jsx]: Fixed. Good eye!

**Pull request merged**